### DISCUSSION TITLE
ReferenceError: AbortController is not defined at fetchCSSFromGoogleFonts

### DISCUSSION THREAD

**Discussion by rishipurwar1** [vercel/next.js#48371] [Help]
### Summary

Hello there,

Yesterday, I upgraded to next 13.3 from 13.1.6, and migrated my Google fonts to next/font/google. However, I am getting an error while building on Vercel. I have already checked the changes in the production build by running npm run build and npm run start and it's working fine, but it's failing on vercel.

```
import { Inter } from "next/font/google"

const inter = Inter({
  subsets: ["latin"],
  variable: "--font-inter",
})

export default function MyApp({ Component, pageProps }) {
  return (
        
          <div className={`${inter.variable} font-sans`}>
            {renderWithLayout(<Component {...pageProps} />)}
          </div>
  )
}
```

Error message:

![Screenshot_2023-04-13_at_8 01 06_AM](https://user-images.githubusercontent.com/57035408/231946285-d3ea2026-b442-4a9c-a358-d07a5325eb16.png)

I'm not using AbortController anywhere in my code.
Anyone please help me with this!

**Comment by elrumordelaluz** [comment]
I am getting the same error, only when deploying to Vercel; `yarn build` in local environment works as expected.

![Screenshot 2023-04-14 at 09 38 55](https://user-images.githubusercontent.com/784056/231976820-2bd889a6-cd3f-4e54-89a4-77062da5e4a8.png)

**Comment by elrumordelaluz** [comment]
Seems to be an issue related on the refactoring released in `v13.2.4`.

I opened the issue #48377. 

Temporarily you can downgrade into `v13.2.3` which works as expected.

**Reply by rishipurwar1** [reply]
okay, thanks!

**Comment by markudevelop** [comment]
require abortcontroller on the clientside (Browser) only should help fix the issue
